Fix the user switch loading issue

ceiba1985 opened the pull request.

sysopenci commented:
Improper Commit Message
Tracked on not found in commit message,
    make sure Tracked-On: Jira-ticket  is present.

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

ceiba1985 committed:
Fix the user switch loading issue

Signed-off-by: jizhenlo <zhenlong.z.ji@intel.com>

sysopenci commented:
Android CI has completed Engineering Build for this issue, build is SUCCESS.To merge the changes please click on APPROVE FOR MERGE button in Android CI WEB

ceiba1985 committed:
Merge branch 'projectceladon:celadon/s/mr0/master' into celadon/s/mr0/master

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

ceiba1985 committed:
Let user build run in enforcing mode

Remove the patch that let user build run in permissive
mode.

Tracked-On: OAM-112154
Signed-off-by: jizhenlo <zhenlong.z.ji@intel.com>

sysopenci commented:
Android CI has completed Engineering Build for this issue, build is FAILURE. Please check the linked Tracked-On issue/Android CI Web  for more details

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed Engineering Build for this issue, build is FAILURE. Please check the linked Tracked-On issue/Android CI Web  for more details
